File: II - Linguaggi di programmazione e laboratorio/Python/Exercises/06_compito_java_20150917/FuoriCasa.py
from Partita import Partita


class FuoriCasa(Partita):
    def __init__(self, tipo, citta, data, ora, risultato, vinta):
        super().__init__(tipo, data, ora, risultato, vinta)
        self._citta = citta

    def get_citta(self):
        return self._citta

    def stampa_partita_trovata(self):
        return self._citta + "\t" + super().__str__()

    def __str__(self):
        return "-\t" + self._citta + "\t" + super().__str__()

File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/06_compito_java_2015_09_17/FuoriCasa.java
import java.util.*;

public class FuoriCasa extends Partita {
    private String citta;

    public FuoriCasa(String data, String ora, String risultato, int vittoria, String citta) {
        super(data, ora, risultato, vittoria);
        this.citta = citta;
    }

    public String toString() {
        return "-\t" + citta + "\t" + data + "\t" + ora + "\t" + risultato;
    }

    public String getCitta() {
        return this.citta;
    }
}